framework/decode/file_processor.cpp

```diff
@@ -507,7 +511,7 @@ void FileProcessor::HandleBlockReadError(Error error_code, const char* error_mes
     }
     else
     {
-        GFXRECON_LOG_ERROR("%s", error_message);
+        GFXRECON_LOG_ERROR("%s (frame %lu block %lu)", error_message, current_frame_number_, block_index_);
```

Comment:
Format codes as per my other comment.

Reply:
Another reason for stepping up to C++ 20 with its modern text formatting.

Reply:
Resolved same was as above.